pkgs/top-level/all-packages.nix

```diff
@@ -370,6 +370,8 @@ with pkgs;
 
   acbuild = callPackage ../applications/misc/acbuild { };
 
+  ac = callPackage ../tools/system/ac { };
```

Comment:
Short names like `ac` at top-level should be reserved for old well-known Unix utilities. I think it's better to call it `fastcat` or `ats-cat`. Also, it should be under `../tools/misc`, like `coreutils`.